changelog.md

```diff
@@ -1,5 +1,15 @@
 ## Changes to 1.8.xxx
 
+### `--gpu_scheduling_behavior` default is now `restricted`; `undefined` is deprecated and will be removed
+
+The default GPU Scheduling Behavior has been changed to `restricted`, and `undefined` has been deprecated and will be removed in `1.9.x`. Operators with GPU clusters that are upgrading to Marathon 1.8.x should think carefully about their desired policy and set accordingly; as a general rule:
+
+* If Marathon is not GPU-enabled (`--enable_features gpu_resources`), you don't need to do anything.
```

Comment:
this sentence is challenging to read because it is a double negative... if not enabled, don't do anything. 
how about "Setting GPU scheduling behavior is only possible if the GPU feature is enabled (`--enable_features gpu_resources`)"